projects/storefrontlib/src/cms-components/user/register/register.component.html

```diff
@@ -203,7 +203,6 @@
           <button
             type="submit"
             (click)="submit()"
-            [disabled]="shouldDisableRegisterButton()"
```

Comment:
This is bringing trouble, we can now submit an invalid registration form, which results in backend errors. I tried to understand #4534, but I couldn't.

Reply:
The backend should return errors when invalid data is submitted. Submission prevention should be handled using the same method as above. Forms have inconsistently handled invalid feedback so it doesn't work the same across the board for every form. Most of these have been brought up as a11y bugs by qa in new tickets and are rather improvements as opposed to regressions.

Reply:
Currently we indeed can send 'empty' requests to the backend. But we already have a plan to fix it with the same implementation as with `product-reviews` (seen in your comment above)